### DISCUSSION TITLE
Transfer between accounts shows as income in report

### DISCUSSION THREAD

**Discussion by dballagi** [firefly-iii/firefly-iii#8341] [Firefly III] [answered]
First of all, let me express my gratitude for this excellent tool, it helps a lot to manage my finances. 

Now that I have a few months of data in the it, I started playing with reports to monitor my spending. Today I noticed that simple transfers between accounts show up as income in the `Revenue / income` section of a generated report (a monthly report in this exact case).
Is there a way to exclude transfers from this section and only regard deposits to asset accounts as income?

Thank you in advance!

**Comment by JC5** [comment, marked as answer]
Add the account to the report. The transfers will disappear.
